doc/source/release-notes/Percona-Server-5.6.38-83.0.rst

```diff
@@ -0,0 +1,54 @@
+.. rn:: 5.6.38-83.0
+
+============================
+|Percona Server| 5.6.38-83.0
+============================
+
+Percona is glad to announce the release of |Percona Server| 5.6.38-83.0 on
+December 8, 2017 (Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.6/Percona-Server-5.6.38-83.0/>`_
+and from the :doc:`Percona Software Repositories </installation>`).
+
+Based on `MySQL 5.6.38
+<http://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-38.html>`_, including
+all the bug fixes in it, |Percona Server| 5.6.38-83.0 is the current GA release
+in the |Percona Server| 5.6 series. All of |Percona|'s software is open-source
+and free, all the details of the release can be found in the `5.6.38-83.0
+milestone at Launchpad
+<https://launchpad.net/percona-server/+milestone/5.6.38-83.0>`_.
+
+New Features
+============
+
+ |Percona Server| has implemented :ref:`innodb_fragmentation_count`.
+
+ |Percona Server| has implemented support for :ref:`aio_page_requests`
+ This feature was ported from a *Facebook MySQL* patch.
+
+
+Bugs Fixed
+==========
+
+ Dynamic row format feature to support ``BLOB/VARCHAR`` in ``MEMORY`` tables
+ requires all the key columns to come before any ``BLOB`` columns. This
+ requirement however was not enforced, allowing creating MEMORY tables in
+ unsupported column configurations, which then crashed or lose data in usage.
+ Bug fixed :bug:`1731483`.
+
+ If an I/O syscall returned an error during the server shutdown with
+ :ref:`threadpool` enabled, a mutex could be left locked. Bug fixed
+ :bug:`1702330` (*Daniel Black*).
+
+ After fixing bug :bug:`1668602`, bug :bug:`1539504`, and bug :bug:`1313901`,
+ temporary tables were not logged in the binary log if under ``ROW`` and
+ ``MIXED`` based binary log mode. Bug fixed :bug:`1711781`.
+
+ Under concurrent environment running ``ANALYZE TABLE`` could lead to a 
```

Comment:
ANALYZE TABLE and a long running query on the same table, then the new queries get the WfTF state.